pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -486,6 +486,9 @@ type ClusterConfig struct {
 	// +optional
 	VPC *ClusterVPC `json:"vpc,omitempty"`
 
+	// +optional
+	PrivateCluster *PrivateCluster `json:"privateCluster,omitempty"`
```

Comment:
Would be nice to document this field right away

Reply:
Good point. Addressed in the latest commit.